server/persistence/mysql/mysql.go

```diff
@@ -36,6 +36,7 @@ func NewFactory(ctx context.Context, config Config) (persistence.DBFactory, erro
 	}
 
 	if config.RunCreateStmts {
+		log.Println("Run Create Stmts...")
```

Comment:
IIRC I added these when I had no clue what I was doing and trying to get connected to RDS. Maybe we can clean these up?